### PR TITLE
major microsite clean up (mostly deletion)

### PR DESCRIPTION
**Pull request by jebeck**
So here's the deal. Y'all have until the next sprint "limbo" to look at these changes (mostly deletions) and protest any of them. If I get crickets, I'm merging this before Sprint 28 starts, and it will go live at that time. You've been warned, _mwahaha_.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
+Gemfile.lock
```
Comment (darinkrauss) [inline .gitignore]: You are supposed to check the Gemfile.lock in. This ensures that everyone gets the exact same version of the gems.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 
 This is a Tidepool microsite for helping to guide and manage the open source nature of our work.
 
-This is a GitHub Pages site, and the deployed site can be found [here](http://tidepool-org.github.io).
+This is a GitHub Pages site, and the deployed site can be found [here](http://developer.tidepool.io/).
 
-If you want to run it locally, you should have a Ruby installation working.
+If you want to run it locally, you should have a Ruby installation working and follow [GitHub's instructions for installing Jekyll](https://help.github.com/articles/using-jekyll-with-pages/#installing-jekyll).
 
-Then do ```gem install bundle``` from within this repository.
```
Comment (darinkrauss) [inline README.md]: Does the Jekyll install above negate the need to install bundler and all the rest of the gems?

Comment (jebeck) [inline README.md]: The reference to GitHub's instructions you mean? No, not really. GitHub's instructions cover things like "1) Install Ruby"

**Comment by darinkrauss** [comment]
Lots of deletes, but I guess that is a start in the right direction of getting accurate and up-to-date information in there.

**Pull request merged**